docs/docs/migrate/keptnapp/index.md

```diff
@@ -53,11 +62,45 @@ Let's say we have the following `KeptnApp` in our cluster:
 {% include "./assets/keptnapp.yaml" %}
 ```
 
-Applying the migration steps from the previous subsection, we get the
-following result:
+Applying the migration steps from the previous subsection,
+we get the following result.
+You see the following changes:
+
+* The `app.kubernetes.io/managed-by` annotation
+  has been added to the `labels` section of the revised `KeptnApp` resource.
+
+     If your original `KeptnApp` resource was auto-generated,
+     it already had this annotation.
+
+* The list of tasks and evaluations
+has been moved from the `KeptnApp` resource
+to the `KeptnAppContext` resource
 
 ```yaml
 {% include "./assets/keptnapp-migrated.yaml" %}
 ```
 
 These modified resources can be now applied to your cluster.
+
+## What's next?
+
+Making these modifications allows your existing `KeptnApp` functionality
+run as it did before.
```

Comment:
this sentence sounds strange 🤔 maybe something stating these changes do not alter Keptn's behavior?

Reply:
I reworded it.